lib/stripe/api_operations/request.rb

```diff
@@ -47,6 +48,14 @@ def request(method, url, params = {}, opts = {})
           end
         end
 
+        private def error_on_invalid_params(params)
+          return if params.is_a?(Hash)
```

Comment:
Shouldn't this check for `nil` too, since we accept it as a valid value?

```suggestion
          return if params.nil? || params.is_a?(Hash)
```

Reply:
Whoops, yeah there should be a `nil`check there. I'll add a spec for that, too.

Reply:
@ob-stripe Actually, it looks like passing `nil` breaks `warn_on_opts_in_params` since it assumes params will respond to `#key?`.

I can add something like `params ||= {}` before the error/warn checks?

Reply:
Yeah, that sounds reasonable.

Reply:
Cool, just pushed up the addition with a regression spec.